src/configure.ac

```diff
@@ -853,6 +859,9 @@ AC_SUBST([AESNI_ARCH],["$aesni_arch"])
 
 aesni_os=""
 aesni_options="DISABLED"
+
+if test "x$simd" = "xyes" ; then
+
```

Comment:
Apparently you are ignoring what I said about AES-NI. I'm willing to discuss the issue but as long as you silently ignore me I will presume you just didn't get my message.

--disable-SIMD should *not* mess with the AES-NI stuff.

Reply:
Ok, I will fix.  No, I did not understand your message until here, now I do.